BookingService/BookingService.Domain/Entities/Booking.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace BookingService.Domain.Entities
+{
+    public class Booking
+    {
+        public int Id { get; set; }
+        public string HostId { get; set; }

```

Comment:
What is Host and what is Tenant?
Why do you need Host id if you can get it using `Apartment.HostId`?